db/fdb_fend.c

```diff
@@ -4340,6 +4350,7 @@ int fdb_trans_commit(sqlclntstate *clnt, enum trans_clntcomm sideeffects)
             if (tran->nwrites) {
                 /* handle is only created upon first remote write to this fdb */
                 assert(tran->fcon.hndl);
+                _fdb_client_set_identityBlob(clnt, tran->fcon.hndl);
```

Comment:
shouldn't we send this when "begin" is sent?

Reply:
We have to send it with each query as identity can change if it's a long transaction.